### PR TITLE
feat:MEDU-167 add datamatrices to pdf

### PR DESCRIPTION
**Pull request by BeatrizRCorreia**

### CODE REVIEW

File: src/main/java/health/medunited/model/LetterRequest.java [hunk 1]
```diff
@@ -1,16 +1,19 @@
 package health.medunited.model;
+import java.util.ArrayList;
 
 public class LetterRequest {
     private String contactName;
     private String contactEmail;
     private String contactMessage;
-    private String attachment;
+    private ArrayList<String> attachment;
```
Comment (simoneS96) [inline src/main/java/health/medunited/model/LetterRequest.java:8]: I would call it attachments with a final "s", as it is an array

File: src/main/java/health/medunited/service/EmailService.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import java.lang.reflect.Array;
```
Comment (simoneS96) [inline src/main/java/health/medunited/service/EmailService.java]: This is not used. It can be deleted

**Review by simoneS96 — APPROVED** [review]

**Comment by simoneS96** [comment]
Looks good. I just left two comments

File: src/main/java/health/medunited/service/EmailService.java (hunk 2 above)
Comment (BeatrizRCorreia) [reply src/main/java/health/medunited/service/EmailService.java]: done

File: src/main/java/health/medunited/model/LetterRequest.java (hunk 1 above)
Comment (BeatrizRCorreia) [reply src/main/java/health/medunited/model/LetterRequest.java:8]: will change this later as the other two repositories have this name

**Pull request merged**